### ISSUE TITLE
Tuple used as immutable sequence is considered imprecise

### ISSUE THREAD

**Issue by mthuurne** [python/mypy#7215]
When running mypy 0.720 on:
```py
from typing import Sequence, Tuple

a: Tuple[int, ...] = tuple(range(10))
b: Sequence[int] = tuple(range(10))
c: Sequence[int] = list(range(10))
```
and producing an HTML report, the definitions for `a` and `b` are marked in yellow, because mypy is uncertain about their types. The module is also reported as "40% imprecise" in the report index.

I don't think there is anything imprecise about these types though. And in case `c`, when using `list` instead of `tuple`, mypy does consider it precise.

**Comment by JukkaL** [comment]
Yes, this looks like a bug.

**Comment by TH3CHARLie** [comment]
No longer reproducible in 0.761(confirmed that happened in 0.720), now the report shows 0.0% imprecise, looks like some PR has fixed this. @JukkaL

**Comment by mthuurne** [comment]
Indeed this doesn't happen anymore on 0.761. I'll close the issue.
